src/test/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/mapper/OffenderMapperTest.java

```diff
@@ -128,4 +128,15 @@ public void shouldMapRequirementDetailsToRequirement() {
         assertThat(requirement3.getTerminationDate()).isEqualTo(LocalDate.of(2018,12,01));
         assertThat(requirement3.getRqmntTerminationReasonId()).isEqualTo("2500052884");
     }
+
+    @Test
+    public void shouldMapEmpty() throws IOException {
+        var objectMapper1 = new ObjectMapper();
+        objectMapper1.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        objectMapper1.enable(DeserializationFeature.ACCEPT_EMPTY_ARRAY_AS_NULL_OBJECT);
```

Comment:
Is this right? I'd expect this to do the exact opposite of what we want and turn a valid empty array into a null? Also I can't see this configured globally anywhere so will this actually be applied in the application runtime?

Reply:
Sorry - that was a hangover from earlier tests. Works without. Actually - in that test class, all 3 files get loaded for all 4 tests, but they're only needed once in each.. Would like to move the file loading into the test methods ? Would be more performant and fewer class fields.